man/journald.conf.xml

```diff
@@ -368,11 +368,13 @@
         <term><varname>ForwardToKMsg=</varname></term>
         <term><varname>ForwardToConsole=</varname></term>
         <term><varname>ForwardToWall=</varname></term>
+        <term><varname>ForwardToSocket=</varname></term>
 
         <listitem><para>Control whether log messages received by the journal daemon shall be forwarded to a
-        traditional syslog daemon, to the kernel log buffer (kmsg), to the system console, or sent as wall
-        messages to all logged-in users.  These options take boolean arguments. If forwarding to syslog is
-        enabled but nothing reads messages from the socket, forwarding to syslog has no effect. By default,
+        traditional syslog daemon, to the kernel log buffer (kmsg), to the system console, sent as wall
+        messages to all logged-in users or sent over a socket. These options take boolean arguments except
+        for <literal>ForwardToSocket=</literal> which takes an an address instead. If forwarding
```

Comment:
please mention briefly what the format is, and provide a link to the description of the format.

might also be good to add a comment somewhere that writing this data out is synchronous, and people need to take care that this isn't slow. Also maybe say that using IP for this is not great in the general case, because typically available too late